### DISCUSSION TITLE
怎么把目录标题黑体改为宋体？

### DISCUSSION THREAD

**Discussion by ZiyeHu** [stone-zeng/fduthesis#295] [Q&A] [answered]
请问怎么把目录和章节的黑体改为宋体？

**Comment by sikouhjw** [comment, marked as answer]
https://github.com/stone-zeng/fduthesis/blob/c4dc227f7c61ea31e9a4af615fa10e3f85618618/source/fduthesis.dtx#L3843-L3856
将 `\sffamily` 删掉。

请确认你没有用错模式，`cn` 模式里就是黑体，`en` 模式才是『宋体加粗』。

**Comment by ZiyeHu** [comment]
嗯嗯，谢谢啦，改成宋体了。还有一个问题是，我看学校发的模板“学术学位”这些字样是宋体15号，但是latex模板是14号字体，怎么改成15号呢？

在2023-03-21 ***@***.***写道：

https://github.com/stone-zeng/fduthesis/blob/c4dc227f7c61ea31e9a4af615fa10e3f85618618/source/fduthesis.dtx#L3843-L3856
将 \sffamily 删掉。

请确认你没有用错模式，cn 模式里就是黑体，en 模式才是『宋体加粗』。

**Reply by sikouhjw** [reply]
你觉得格式审查会这么细吗？为什么之前的人没有被拦下来？

**Comment by ZiyeHu** [comment]
我师兄说去年因为latex模板被外审专家为难了几个人被毙掉了

在2023-03-21 ***@***.***写道：

你觉得格式审查会这么细吗？为什么之前的人没有被拦下来？

**Reply by sikouhjw** [reply]
1. 问题在于，被毙掉不代表跟模板有关，不代表跟撰写工具有关
2. 如果你认为与模板有关，请你拿出切实的证据，来让模板作者修改模板
3. issue 和 discussion 没有关于这件事的反馈，如果真是因为模板被毙掉，我认为会向模板作者反馈
